tests/e2e/steps/steps_def.go

```diff
@@ -1396,287 +1395,17 @@ func (step *Step) configureAuthorizationProxyServer(res Resource, driver string,
 	}
 
 	address := proxyHost
-	// For v1.9.1 and earlier, use the old address
-	configVersion := cr.GetModule(csmv1.AuthorizationServer).ConfigVersion
-	isOldVersion, _ := utils.MinVersionCheck(configVersion, "v1.9.1")
-	if isOldVersion {
-		address = "authorization-ingress-nginx-controller.authorization.svc.cluster.local"
-	}
-
 	fmt.Printf("Address: %s\n", address)
 
+	configVersion := cr.GetModule(csmv1.AuthorizationServer).ConfigVersion
```

Comment:
Do we still need this switch statement if v1 is removed from e2e?

Reply:
Well, technically no, it's not needed. I just left it for future support if we ever needed, but we can remove the switch. Fixed

Also, it seems we don't need the `configVersion` at all. Using of this param was removed from AuthorizationV2Resources() a long time ago. I removed it as well